eng/DotNetBuild.props

```diff
@@ -1,56 +1,23 @@
 <!-- When altering this file, include @dotnet/product-construction as a reviewer. -->
 
 <Project>
+  <PropertyGroup>
+    <!-- TargetRid can be set by the orchestrator, so we need to set OutputRID to that as a default. -->
+    <OutputRID Condition="'$(OutputRID)' == ''">$(TargetRid)</OutputRID>
+  </PropertyGroup>
```

Comment:
As mentioned on the other PR, OutputRID is the runtime's name for TargetRid and these may probably be squashed.

This could be deferred to a separate PR.

Reply:
I think I'll do that in a separate PR to try to keep this PR more self-contained (as we use OutputRID all across the repo).